### PR TITLE
Adds filters for AstropyDeprecationWarning for 0.8

### PR DESCRIPTION
**Pull request by yashkgp**
Added a warning filter to tackle `AstropyDeprecationWarning` in `sunpy.coordinates.representation` in Sunpy 0.8 .

### CODE REVIEW

**Comment by nabobalis** [comment]
I just merged this locally and the warning still happens. 
```
>>> from sunpy import coordinates
WARNING: AstropyDeprecationWarning: 
The 'recommended_units' attribute is deprecated since 3.0 and may be removed
in a future version. Its main use, of representing angles in degrees in frames,
is now done automatically in frames. Further overrides are discouraged but can
be done using a frame's ``frame_specific_representation_info``.
 [astropy.coordinates.representation]
```

File: sunpy/coordinates/representation.py [hunk 1]
```diff
@@ -117,3 +120,7 @@ class SphericalWrap180Representation(SphericalRepresentation):
 
     _unitrep = UnitSphericalWrap180Representation
 
+
+with warnings.catch_warnings():
+  warnings.simplefilter('ignore', AstropyDeprecationWarning)
+
```
Comment (nabobalis) [inline sunpy/coordinates/representation.py]: I am not sure that this is the correct way. But it isn't the right place in the script for this.

Comment (yashkgp) [reply sunpy/coordinates/representation.py]: Hey @nabobalis , Mistakenly I was testing on older version of Astropy, that why I was not getting a warning on testing , hopefully now I fixed the problem .

**Comment by nabobalis** [comment]
This seems to work now. We have some test issues but I think they are unrelated to the PR

**Pull request merged**